### PR TITLE
Special metadata for special day!

### PR DESCRIPTION
**Pull request by pllim**
### Description

The metadata you didn't know you needed.

Original query credit goes to @eteq .

Screenshot 2022-04-01 083453

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [x] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Review by kecnry — CHANGES_REQUESTED** [review]
Needs more test coverage, which might be difficult.  Perhaps we should build a database-server with a user-voting system to track the predictiveness of the futures.  Overtime, we could even integrate machine learning for a better and swap out the existing URL call with our own web API for a better user-experience.

File: jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py [hunk 1]
```diff
@@ -15,9 +15,14 @@ class MetadataViewer(TemplateMixin):
...
     has_metadata = Bool(False).tag(sync=True)
     metadata = List([]).tag(sync=True)
+    zodiac_items = List(["Aries", "Taurus", "Gemini", "Cancer", "Leo", "Virgo", "Libra", "Scorpio",
+                         "Sagittarius", "Capricorn", "Aquarius", "Pisces"]).tag(sync=True)
+    selected_zodiac = Unicode("").tag(sync=True)
+    your_future = Unicode("").tag(sync=True)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self._cached_futures = {}
```
[the remark this answers is not in the document]
Comment (pllim) [reply jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py:25]: You know, I thought about cache invalidation but I do not dare defy the voice of the stars.

Good idea though, and I think is doable if we include timestamp. But can get tricky if user did this, say, in an airport, and then run this again in a completely timezone after Taurus prediction asked them to go to Nepal or something.

**Comment by pllim** [comment]
Well, that was fun. Thanks for the review! 😸

**Comment by havok2063** [comment]
I think you should make this permanent in the code but hidden.  And schedule it to only be visible in the UI on April 1st every year.

**Comment by rosteen** [comment]
> I think you should make this permanent in the code but hidden. And schedule it to only be visible in the UI on April 1st every year.

I was a little sad when that turned out not to be the case...but we don't want to end up with errors due to deprecated joke code, sadly 😆

**Comment by pllim** [comment]
Lemme just say that the astrology web service been working for many years... 😆 

Also if @kecnry can figure out how to make plugins be time based, I am all for it. 😉

**Pull request closed without merging**